### PR TITLE
Created "Setting up an Android Development environment on Compute Engine" Tutorial

### PR DESCRIPTION
**Pull request by Rishit-dagli**
Created "Setting up an Android Development environment on Compute Engine" Tutorial. 

## Overview of the tutorial

This tutorial shows how to set up an [Android Studio](https://developer.android.com/studio) on Google Cloud Platform (GCP) in just a few minutes. Follow this tutorial to configure an Android Studio development environment on an Ubuntu or Windows Server virtual machine instance on Compute Engine with the ability to use Android Emulators and accelerate your development.

We will set up a full-fledged environment which is fast enough to do Gradle Builds in a second!

### CODE REVIEW

**Comment by ToddKopriva** [comment]
Thanks for the submission. There are a few other items in the review queue, so it will be a few days before this review is done.

**Comment by ToddKopriva** [comment]
I'll begin an editorial and production review on this tomorrow.

File: tutorials/setting-up-an-android-development-environment-on-compute-engine/index.md [hunk 1]
```diff
@@ -32,11 +32,6 @@ This tutorial uses billable components of Google Cloud, including Compute Engine
 Use the [pricing calculator](https://cloud.google.com/products/calculator/) to generate a cost estimate based on your projected usage. New Google Cloud
 users might be eligible for a [free trial](https://cloud.google.com/free-trial).
 
-If you follow this tutorial, the costs are estimated as follows, depending on the operating system that you use:
-
-* [Linux VMs](https://cloud.google.com/products/calculator#id=ca135004-465c-4a43-bc5b-701af07df644)
```
Comment (Rishit-dagli) [inline tutorials/setting-up-an-android-development-environment-on-compute-engine/index.md]: @ToddKopriva The following link works for me, could you do a check again.

File: tutorials/setting-up-an-android-development-environment-on-compute-engine/index.md [hunk 2]
```diff
@@ -32,11 +32,6 @@ This tutorial uses billable components of Google Cloud, including Compute Engine
...
-* [Windows VMs](https://cloud.google.com/products/calculator#id=eff1ebe1-1ed8-4475-a5f2-d07519fb5883)
```
Comment (Rishit-dagli) [inline tutorials/setting-up-an-android-development-environment-on-compute-engine/index.md]: @ToddKopriva The following link works for me, could you do a check again.

File: tutorials/setting-up-an-android-development-environment-on-compute-engine/index.md (hunk 1 above)
Comment (ToddKopriva) [reply tutorials/setting-up-an-android-development-environment-on-compute-engine/index.md]: Yes, I'll check again as I make my second pass.

File: tutorials/setting-up-an-android-development-environment-on-compute-engine/index.md (hunk 2 above)
Comment (ToddKopriva) [reply tutorials/setting-up-an-android-development-environment-on-compute-engine/index.md]: Yes, I'll check again as I make my second pass.

Comment (ToddKopriva) [reply tutorials/setting-up-an-android-development-environment-on-compute-engine/index.md]: I just checked, and the Linux link does go to a pricing calculator that shows a Linux OS, but the Windows link does not go to a pricing calculator that shows Windows Server.

Please re-create the pricing calculator links and modify the pull request with the new links.

**Comment by ToddKopriva** [comment]
@Rishit-dagli , I have finished my editing and production pass.

I took out a lot of material that was copied from the Google Cloud documentation, and I replaced that material with links to the official documentation. This is mostly so that the information in your tutorial doesn't become stale when such things as the supported operating systems and other details change.

Let me know whether you want to make additional changes or discuss the edits that I made. As far as I'm concerned, this is ready to merge now, but I'll leave that up to you.

File: tutorials/setting-up-an-android-development-environment-on-compute-engine/index.md (hunk 2 above)
Comment (Rishit-dagli) [reply tutorials/setting-up-an-android-development-environment-on-compute-engine/index.md]: @ToddKopriva Thanks for letting me know about that, it must be a mistake from my end, I will fix it.

**Comment by Rishit-dagli** [comment]
@ToddKopriva I have added the updated prefilled pricing calculator links now.

**Comment by ToddKopriva** [comment]
> @ToddKopriva I have added the updated prefilled pricing calculator links now.

Thanks.

**Comment by ToddKopriva** [comment]
I've merged the pull request. I'll publish this new tutorial on Monday to cloud.google.com/community .

**Pull request merged**